### ISSUE TITLE
Issue when Klarna denied order

### ISSUE THREAD

**Issue by JoostWan** [buckaroo-it/Magento2#40]
When Klarna denied an order there is an error in the code: https://github.com/buckaroo-it/Magento2/blob/master/Observer/HandleFailedQuoteOrder.php#L64

```
[28-Jul-2020 14:41:35 UTC] PHP Fatal error:  Uncaught Error: Call to undefined method Buckaroo\Magento2\Model\Plugin\Method\Klarnakp::save() in /Users/username/Sites/client/vendor/buckaroo/magento2/Observer/HandleFailedQuoteOrder.php:65
Stack trace:
#0 /Users/username/Sites/client/vendor/magento/framework/Event/Invoker/InvokerDefault.php(88): Buckaroo\Magento2\Observer\HandleFailedQuoteOrder->execute(Object(Magento\Framework\Event\Observer))
#1 /Users/username/Sites/client/vendor/magento/framework/Event/Invoker/InvokerDefault.php(74): Magento\Framework\Event\Invoker\InvokerDefault->_callObserverMethod(Object(Buckaroo\Magento2\Observer\HandleFailedQuoteOrder), Object(Magento\Framework\Event\Observer))
#2 /Users/username/Sites/client/vendor/magento/framework/Event/Manager.php(66): Magento\Framework\Event\Invoker\InvokerDefault->dispatch(Array, Object(Magento\Framework\Event\Observer))
#3 /Users/username/Sites/client/generated/code/Magento/Framework/Event/Manager/Proxy.php(95): Magento\Framework\Event\Manager->dispatch('sales_model_ser...', Array)
#4 /Users/username/Sites/client/vendor in /Users/username/Sites/client/vendor/buckaroo/magento2/Observer/HandleFailedQuoteOrder.php on line 65
```

We changed it to the following code and then it’s working fine. The order is being canceled and saved.
```
$order->cancel();
$order->save();
```

Can you also add the error message from Buckaroo/Klarna as a comment to the order with `$order->addStatusHistoryComment()` so the shop owner can see what the reason is for the canceled order.

**Comment by vladislav-padalka-hys** [comment]
Thanks for your report !
It was added to the master branch : 
https://github.com/buckaroo-it/Magento2/commit/40f4a3dba3e8447ae9b51520649b1357466e9d00
https://github.com/buckaroo-it/Magento2/commit/57039822d72ac0f9bc6ce53744de3614e1fcf133

**Comment by Buckaroo-Rens** [comment]
Fix included in latest release 1.25.2
